conftest.py

```diff
@@ -154,13 +158,22 @@ def _add_upgrade_test(_item: Item, _upgrade_deployment_modes: list[str]) -> bool
     post_upgrade_tests: list[Item] = []
     non_upgrade_tests: list[Item] = []
     upgrade_deployment_modes: list[str] = []
+    formatted_product_version: Version | None = None
 
     run_pre_upgrade_tests: str | None = config.getoption(name="pre_upgrade")
     run_post_upgrade_tests: str | None = config.getoption(name="post_upgrade")
     if config_upgrade_deployment_modes := config.getoption(name="upgrade_deployment_modes"):
         upgrade_deployment_modes = config_upgrade_deployment_modes.split(",")
 
+    if not session.config.option.setupplan and not session.config.option.collectonly:
+        if product_version := get_product_version(admin_client=get_client(), raise_on_missing_csv=False):
```

Comment:
Why don't you want to raise on missing csv?

Reply:
because on upstream you will not have it;
i have another pr that i am working on for cluster sanity - there i will check if this is an odh/rhoai deployment and raise